patches/0004-Add-OpenSSL-crypto-backend.patch

```diff
@@ -191,12 +194,72 @@ index 00000000000000..e6695dd66b1d02
 +
 +var Enc = bbig.Enc
 +var Dec = bbig.Dec
+diff --git a/src/crypto/internal/backend/fips140/nosystemfips.go b/src/crypto/internal/backend/fips140/nosystemfips.go
+index 9e12c6d5d2239c..06b08be3ce5106 100644
+--- a/src/crypto/internal/backend/fips140/nosystemfips.go
++++ b/src/crypto/internal/backend/fips140/nosystemfips.go
+@@ -2,6 +2,8 @@
+ // Use of this source code is governed by a BSD-style
+ // license that can be found in the LICENSE file.
+ 
++//go:build !goexperiment.opensslcrypto
```

Comment:
Could this be `!goexperiment.systemcrypto`?

(Even if a backend doesn't implement `systemFIPSMode`, it'd be less prone to conflicts to have each one add its own file that also simply returns false.)

Reply:
Agree with both. Will do 👍.